src/common/entity/alarm_panel_icon.ts

```diff
@@ -0,0 +1,21 @@
+/** Return an icon representing a alarm panel state. */
+
+export const alarmPanelIcon = (state?: string) => {
+  switch (state) {
+    case "armed_away":
+    case "armed_vacation":
+      return "hass:shield-lock";
+    case "armed_home":
+      return "hass:shield-home";
+    case "armed_night":
+      return "hass:shield-moon";
+    case "armed_custom_bypass":
+      return "hass:security";
+    case "pending":
+      return "hass:shield-outline";
+    case "triggered":
+      return "hass:bell-ring";
+    default:
+      return "hass:shield-check";
```

Comment:
This should be the `disarmed` state I think, or `shield-off`? We should add a different default, I think, maybe just `shield`.